### PR TITLE
Updating Pets at home logo

### LINKED ISSUE #9139: Update: Pets at Home
### Brand Name

Pets at Home

### Official Resources for Icon and Color

https://media.petsathome.com/wcsstore/pah-as01/images/redesign/new-logo.svg

https://github.com/simple-icons/simple-icons/issues/7164

https://identitydesigned.com/pets-at-home/

### Additional Comments

We recently rebranded and I forgot to add this

### Contributing

Yes

### PR DESCRIPTION
**Pull request by userbradley**
**Issue:** closes #9139

**Similarweb rank:**
Global: 11,009

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Updated the Pets at Home logo using the latest icon from the site

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9189,7 +9189,7 @@
         },
         {
             "title": "Pets at Home",
-            "hex": "4BA840",
+            "hex": "04A928",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:9192]:
```suggestion
            "hex": "00AA28",
```
Color picked from wordmark was different

File: icons/petsathome.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...-89.2 66.8 0 30.3 22 53 60.2 62.5l21.4 5.3c27.3 6.6 30.6 12.8 30.6 22.4 0 11.2-9.5 18.8-25.3 18.8-25 0-43.4-9.2-63.8-29.6-3-3-4.9-4.9-8.5-4.9-3.3 0-5.9 1.3-7.9 4.6-12.5 20.7-22 28.3-35.2 28.3-14.1 0-24-9.9-24-28v-95.1h31c4.1 0 7.5-3.4 7.5-7.5V89.1c0-4.1-3.4-7.5-7.5-7.5h-31V42.8c0-8.5-5.6-11.8-14.1-8.2l-47.4 19.8c-3.6 1.6-5.6 4.6-5.6 8.5v92c0 57.6-38.5 96.7-87.5 96.7-25.7 0-49-9.9-57.6-34.9 5.9 2.3 15.8 3.3 29.9 3.3 56.6 0 96-30.9 96-75.7 0-39.8-32.9-66.1-81.9-66.1-18.2 0-35.7 3.5-50.8 9.9C280.3 34.9 228.6 0 157.2 0 124.6 0 95 4 67.7 12.2c-7.6 2.3-12.2 7.2-12.2 14.8v129.9C39 150.3 26.5 140.5 16 127c-2.3-3-4-4.9-7.6-4.9-5.3 0-8.5 4-8.5 9.2 0 22.4 26.3 52 55.6 66.5v89.5c0 4.1 3.4 7.5 7.5 7.5h58.7c4.1 0 7.5-3.4 7.5-7.5v-67.8c4.3.3 8.5.7 13.2.7 38.8 0 68-6.1 89.3-19.9C236 259 279.9 300 346.9 300c34.5 0 75.3-12.2 98-43.1 10.9 26.3 36.5 43.1 67.8 43.1s50-7.9 65.8-26c20.4 17.1 43.1 26 73.4 26 54 0 88.2-27 88.2-69.4-.1-30.3-16.6-50.3-61.3-61.2zm-549.5-2V65.8c0-6.2 2.6-9.9 8.5-10.8 8.9-1.6 14.8-1.6 19.7-1.6 39.8 0 63.5 21 63.5 55.9.1 36.8-28.2 57.2-91.7 58.1zM337 120.7c15.8 0 24.3 9.5 24.3 22.7 0 22-19.7 36.8-49 36.8-10.2 0-16.1-.7-20.1-2.3.1-34.1 17.5-57.2 44.8-57.2z" fill="#00aa28"/></svg>
```
Comment (PratyushJayachandran) [inline icons/petsathome.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Pets at Home</title><path d="M4.283 17.577c0-.756.409-1.232 1.031-1.232a.65.65 0 0 1 .613.389v-.24c0-.068.039-.107.107-.107h.5c.067 0 .106.039.106.107v2.163c0 .068-.039.107-.107.107h-.499c-.068 0-.107-.04-.107-.107v-.237a.65.65 0 0 1-.613.389c-.622 0-1.03-.477-1.03-1.232zm1.19.7c.302 0 .46-.236.46-.7s-.158-.7-.46-.7-.46.236-.46.7.159.7.46.7zm2.305-2.211v.32h.383c.068 0 .107.04.107.108v.337c0 .068-.039.107-.107.107h-.383v1.025c0 .25.094.36.237.36a.49.49 0 0 0 .175-.043c.1-.042.16.01.16.104 0 .237-.27.425-.598.425-.505 0-.687-.318-.687-.756v-1.776c0-.042.023-.072.062-.091l.502-.21c.088-.04.15-.004.15.09zm2.157 2.7h-.532c-.068 0-.107-.038-.107-.106v-2.792c0-.068.039-.107.107-.107h.532c.068 0 .107.04.107.107v1.038h1.047v-1.038c0-.068.039-.107.107-.107h.532c.068 0 .107.04.107.107v2.792c0 .068-.04.107-.107.107h-.532c-.068 0-.107-.04-.107-.107v-1.184h-1.047v1.184c0 .068-.04.107-.107.107zm2.273-1.19c0-.755.46-1.231 1.19-1.231.729 0 1.19.476 1.19 1.232 0 .755-.461 1.232-1.19 1.232-.73 0-1.19-.477-1.19-1.232zm1.186.701c.302 0 .46-.236.46-.7s-.158-.7-.46-.7c-.301 0-.46.236-.46.7s.162.7.46.7zm3.463.49h-.5c-.067 0-.106-.04-.106-.107v-1.453c0-.243-.072-.288-.201-.288a.518.518 0 0 0-.383.172v1.566c0 .068-.039.107-.107.107h-.5c-.067 0-.106-.04-.106-.107v-1.998c0-.042.022-.071.061-.09l.503-.211c.09-.04.15-.004.15.09v.344c.129-.318.359-.45.677-.45.282 0 .49.132.587.428a.646.646 0 0 1 .638-.428c.38 0 .688.207.688.732v1.58c0 .067-.04.106-.107.106h-.5c-.068 0-.107-.039-.107-.107v-1.452c0-.243-.07-.289-.2-.289a.518.518 0 0 0-.383.172v1.566c.003.078-.036.117-.104.117zm1.77-1.213c0-.7.4-1.21 1.19-1.21s1.138.575 1.138 1.204v.065c0 .068-.038.107-.107.107h-1.507c0 .282.227.57.72.57.204 0 .356-.052.55-.12.111-.039.199-.016.199.104 0 .23-.34.535-.928.535-.797 0-1.255-.493-1.255-1.255zm1.65-.272c-.028-.289-.194-.448-.46-.448s-.428.16-.46.448h.92zm1.735-6.598-.661-.16c-.726-.171-.992-.362-.992-.713 0-.353.353-.586.895-.586.619 0 1.076.214 1.835.726.084.052.159.097.214.097.107 0 .181-.042.2-.2.024-.173.033-.332.033-.546 0-.32-.01-.535-.032-.651-.033-.182-.097-.276-.214-.354-.63-.395-1.333-.564-2.11-.564-1.752 0-2.893.853-2.893 2.166 0 .982.713 1.718 1.952 2.026l.694.172c.885.214.992.415.992.727 0 .363-.308.61-.82.61-.811 0-1.408-.3-2.07-.96-.096-.098-.158-.16-.275-.16a.279.279 0 0 0-.256.15c-.405.67-.713.917-1.141.917-.457 0-.778-.32-.778-.908V9.39h1.005c.133 0 .243-.11.243-.243V8.08a.245.245 0 0 0-.243-.243h-1.005V6.579c0-.276-.182-.383-.457-.266l-1.537.642a.288.288 0 0 0-.182.275v2.983c0 1.868-1.248 3.136-2.837 3.136-.833 0-1.589-.321-1.868-1.132.192.075.513.107.97.107 1.835 0 3.113-1.002 3.113-2.454 0-1.29-1.067-2.143-2.656-2.143a4.25 4.25 0 0 0-1.647.32C9.092 6.324 7.415 5.192 5.1 5.192c-1.057 0-2.017.13-2.902.396-.246.074-.395.233-.395.48v4.211a2.963 2.963 0 0 1-1.281-.97C.447 9.213.392 9.15.276 9.15c-.172 0-.276.13-.276.298 0 .727.853 1.686 1.803 2.156v2.902c0 .133.11.244.243.244h1.903c.133 0 .243-.11.243-.244v-2.198c.14.01.276.023.428.023 1.258 0 2.205-.198 2.896-.646.14 1.904 1.563 3.233 3.735 3.233 1.119 0 2.442-.395 3.178-1.397.353.852 1.183 1.397 2.198 1.397s1.621-.256 2.133-.843c.662.555 1.398.843 2.38.843 1.751 0 2.86-.875 2.86-2.25-.003-.983-.538-1.631-1.988-1.984zm-17.816-.065V7.325c0-.202.084-.321.275-.35.289-.053.48-.053.639-.053 1.29 0 2.059.681 2.059 1.813.003 1.193-.914 1.855-2.973 1.884zm6.734-1.514c.512 0 .788.308.788.736 0 .713-.639 1.193-1.589 1.193-.33 0-.522-.023-.652-.075.004-1.105.568-1.854 1.453-1.854z"/></svg>
```
Please apply suggestion or redo using source.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please read [Contribution Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md). I have suggested changes which you may apply or redo from source for the tests to pass and be considered for addition to this project.

File: icons/petsathome.svg (hunk 2 above)
Comment (userbradley) [reply icons/petsathome.svg]: Applied suggesition

File: _data/simple-icons.json (hunk 1 above)
Comment (userbradley) [reply _data/simple-icons.json]: Applied suggestion

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for this change, @userbradley. As suggested by @PratyushJayachandran, it looks like the icon is failing the linter, as you've not followed the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) correctly. Could you please review and give this another go? If the source or colour is different from when we included the previous version, the JSON file will also need updating. Feel free to leave a comment if you need some more support.

**Comment by userbradley** [comment]
Hey @adamrusted - I'm struggling here. Are you able to give me a hint where I've gone wrong?

**Comment by adamrusted** [comment]
Apologies for the delay @userbradley - please see feedback below:

It would appear the icon you've submitted is still two paths, these should be merged in your graphics software (Inkscape is a great free one if you're unsure) - you'll also need to resize the **canvas** to 24*24px - and adjust the size of the icon so that the longest side (in this case, the width) is equal to 24px. Be careful to maintain the aspect ratio of the icon, so we don't break it! If you then save that, and run `npx svgo icons/petsathome.svg` it should do most of the work for you.

If you could also review `_data/simple-icons.json` to ensure it's all up to date, including the source URL and the hex code, and run `npm run lint` on your local machine to check for any errors.

**Comment by adamrusted** [comment]
Did you want a hand on this one @userbradley?

**Comment by userbradley** [comment]
Hey @adamrusted I'll most likely take a look on Monday at work at this one

(I work for Pets At Home)

**Review by PratyushJayachandran — APPROVED** [review]
Perfect match @adamrusted! Thank you for starting the PR @userbradley 🎉

**Pull request merged**